### DISCUSSION TITLE
yiwen-blog-website/docs/web/pnpm_npm_yarn

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#90] [Announcements]
# yiwen-blog-website/docs/web/pnpm_npm_yarn

npm（Node Package Manager）、yarn 和 pnpm 都是 JavaScript 生态系统中流行的包管理工具，用于自动化安装、配置和管理项目依赖。尽管它们的基本目的相同，但它们在性能、特性和工作方式上有一些关键的差异

https://hao-yiwen.github.io/yiwen-blog-website/docs/web/pnpm_npm_yarn

**Comment by Hao-yiwen** [comment]
pnpm在rn使用需要添加额外一些配置。
